package.json

```diff
@@ -16,6 +16,7 @@
     "string-width": "^4.2.0"
   },
   "devDependencies": {
+    "@colors/colors": "1.5.0",
```

Comment:
The `@colors/colors` package is still required for test the legacy `cli-table` package in the `verify-legacy-compatibility-test.js` file, therefore it should be in `devDependencies`